doc/host_create.md

```diff
@@ -228,14 +231,16 @@ Available keys for `--compute-attributes`:
 cluster
 template   # hardware profile to use
 cores      # int value, number of cores
+sockets    # int value, number of sockets
 memory     # amount of memory, int value in bytes
 start      # Must be a 1 or 0, whether to start the machine or not
 ```
 
 Available keys for `--interface`:
 ```
-compute_name         # eg. eth0
-compute_network      # select one of available networks for a cluster
+name         # eg. eth0
+network      # select one of available networks for a cluster
+interface    # interface type
```

Comment:
after the rebase, all the --interface attributes,  it should stay with "compute_" prefix.